### DISCUSSION TITLE
Climate Card Showing unavailable

### DISCUSSION THREAD

**Discussion by Rgn589** [Blackymas/NSPanel_HA_Blueprint#2191] [General]
Hi guys , I was trying to set up the climate card . The integration is my air HAVAC in home assistant . So i have provided the climate.AC as entity . But in Climate card . I am getting the set temperature  and the line graph to select temperature but there is a big heading saying Unavailable on top. As shown below
![NSpanel error](https://github.com/user-attachments/assets/8cfaffe8-e54c-47fa-a128-cb26b29063e6)

Any ideas on what this would be is greatly appreciated.

**Comment by edwardtfn** [comment]
Are you using a custom TFT?

Do you mind sharing the info about your climate entity from Developer Tools > States? Please include both "State" and "Attributes".

**Reply by Rgn589** [reply]
I am not using custom tft. I am using the default us tft . I flashed it from the esphome dropdown option.
![Screenshot_20240719_135412_Home Assistant](https://github.com/user-attachments/assets/86a9d31f-ee61-4c7e-8931-0ca04ce16565)
 This is the attributes
hvac_modes:
  - "off"
  - cool
  - heat
  - fan_only
  - dry
min_temp: 16
max_temp: 32
target_temp_step: 1
fan_modes:
  - low
  - medium
  - high
  - auto
current_temperature: null
temperature: 26
fan_mode: auto
friendly_name: AC
supported_features: 393

Let me know if you need anything more

Regards

**Comment by garrycopland** [comment]
I have the same issue - the problem is the current temperature of the air con unit itself is NULL (see attribute current_temperature: null)
I've done a set state in the yaml to prove that's the attribute the blueprint is looking at.
I guess Unavailable is the nicest way displaying NULL, on the Card. Maybe it could default to the panel temp ?
